content/metrics/metrics-1/contents.lr

```diff
@@ -0,0 +1,14 @@
+title: How is it even possible to count users in an anonymity network?
+---
+key: 1
+---
+description:
+
+We actually don't count users, but we count requests to the directories
+that clients make periodically to update their list of relays and estimate
+user numbers indirectly from there.
+
+---
+seo_slug: metrics
```

Comment:
1. it would be nice to have a short URL like: user-estimation. So you will need to rename this question directory.

2. You should also change the seo_slug (and need to separate with hyphens), for example, user-estimation